xmtp_mls/migrations/2024-12-04-190255_add_dm_id_to_groups/up.sql

```diff
@@ -0,0 +1,6 @@
+ALTER TABLE groups
+ADD COLUMN dm_id TEXT;
+
+UPDATE groups
+SET dm_id = LOWER(CONCAT('dm:', dm_inbox_id))
```

Comment:
@richardhuaaa Do you mind chiming in on if you think this approach is okay versus what you suggested here https://github.com/xmtp/libxmtp/issues/1142

This is a little different than what was suggested of "dm:{lower_inbox_id}:{higher_inbox_id}".to_lower_case() because we don't need this dm_id to be the same for both parties correct? Only for the current inboxId to thread. That being said if we want to go this more verbose route it will greatly complicated the migration.